lib/internal/Magento/Framework/App/Response/HttpInterface.php

```diff
@@ -16,4 +16,49 @@
      * @return void
```

Comment:
Probably. If `clearHeader` is added, then I guess the dependency of the `PageCache` module classes might be changed from the concrete implementation to the interface, too.